src/Controllers/users.js

```diff
@@ -9,6 +9,13 @@ const {
   updateUserById,
 } = require("../DAL/users");
 
+const extractUserProps = (user) => ({
```

Comment:
why is this needed?

Reply:
I don't want to return passwords as part of the user object, maybe there is a better way to extract props in-case the user object changes, but I think it's fine for the scope of this assignment